Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -300,8 +318,53 @@ its :php:`__invoke()` method will be called:
 
 Read :ref:`how to configure dependency injection in extensions <dependency-injection-in-extensions>`.
 
+..  index:: Override Event Listener; Event override
+..  _EventListenerOverride:
+
+Overriding event listeners
+--------------------------
+
+Existing event listeners can be overridden by custom implementations. This can be
+performed via :file:`EXT:some_extension/Configuration/Services.yaml`.
+
+For example, a third-party extension listens on the event
+:php:`\TYPO3\CMS\Frontend\Event\ModifyHrefLangTagsEvent` with the following code:
+
+..  literalinclude:: _ServicesOverrideBase.yaml
+    :language: yaml
+    :caption: EXT:some_extension/Configuration/Services.yaml
+
+If you want to replace this event listener with your custom implementation, your extension can
+achieve this by specifying:
+
+..  literalinclude:: _ServicesOverrideCustom.yaml
+    :language: yaml
+    :caption: EXT:my_extension/Configuration/Services.yaml
+
+..  note::
 
+    When overriding an event listener, be sure to check whatever the
+    listener provided as changes to an event. Your own event listener
```

Comment:
Had to read it multiple times: "to check whatever the listener provided as changes to an event".

What do you want to say?

Reply:
I wanted to indicate that one should look up the extension that you override. See what changes the original extension made to the event data. And then ensure, that your custom event listener implementation of the override still performs the needed manipulations.

How about this instead:

"When overriden an event listener, investigate the original implementation. Ensure that your own event listener still performs all needed modifications to the event data, because the original implementation will no longer take effect. If the original extension for example set a property to a specific value, this is now in your responsibility to manage."